CHANGELOG.md

```diff
@@ -1,9 +1,35 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+This project adheres to [Semantic Versioning](http://semver.org/)
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## 1.2.0 2017-05-13
+## [Unreleased]
+
+## [1.3.0] - 2018-02-04
+### Added
+- Initial support for OpenBSD! (@smbambling)
+- Ubuntu now get's `apt-transport-https` installed to support HTTPS repos. (@kevit)
+- Default to HTTPS APT repo's.  @jaredledvina
+- Ubuntu - Redis now always binds to `0.0.0.0` to ensure it's accessible. (@tculp)
```

Comment:
I'd say that should be under a `### Breaking Changes` header and should outline the security implications of the switch. I would still question that that is a sane default over say `127.0.01`  as it is more common than you think to have servers with multiple nics and some being public. I would never do this in my environment but I have seen this done. This is not the place to debate whether or not this is a good idea, that said I think that we need to clearly warn people and ensure we version and explain it clearly. People who have multiple nics and have public interfaces will want the opportunity to spot and ensure that they choose the right interface ip.

Actually now that I think about it my sensu does have a public ip address as we want to ensure that even if our NAT instances are down alerts still come out. While we are safe by using strict aws security groups that does not mean others are as careful.

While some might argue this does not change the public api this breaks users in a very real way and my opinion semver is only helpful about informing users of risks if you actually take it beyond the letter of the law and think about how this impacts other consumers of your software.

Just to play devil's advocate: this restores a uniform experience as other platforms already had it, I could be talked into putting it under a `### Fixed` header instead.

Reply:
So, the change in question here is, https://github.com/sensu/sensu-ansible/pull/105/files, which updated the Ubuntu config to match what we already had done with Debian: https://github.com/sensu/sensu-ansible/blob/master/tasks/Debian/redis.yml#L13-L18, CentOS: https://github.com/sensu/sensu-ansible/blob/master/tasks/CentOS/redis.yml#L19-L23, FreeBSD: https://github.com/sensu/sensu-ansible/blob/master/tasks/FreeBSD/redis.yml#L12-L17, and OpenBSD: https://github.com/sensu/sensu-ansible/blob/master/tasks/OpenBSD/redis.yml#L12-L17.

So, it does unify what we have done for the other supported operating systems. I totally agree that configuring `0.0.0.0` at all might be sketch, maybe that's something we track as an issue and possibly merge/release as a dedicated breaking change. Solely for the change in this release though, I don't feel it's a breaking change but more-so an update to ensure Ubuntu lines up with everyone else. So, maybe `Fixed` is more accurate. Although, I had considered that and I guess I wasn't sure if anything is really `Fixed` per-se but reading through https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#fixing-a-bug, I can see how this might fit under that title. 

Now that said, while this role supports deploying the full stack, as I mentioned in  https://github.com/sensu/sensu-ansible/issues/118, ideally, for real Production use, folks don't flip the Redis/RabbitMQ bits for this role and use dedicated upstream roles for that. Mostly because this role exposes just the minimum to get both of them up in a sane way versus a dedicate role that will support everything. 

Perhaps I'm way off here, curious what you think @majormoses

Reply:
Sounds good in this case to use `### Fixed` I'd still like to make sure we call out the security implications and the fact that it did change in a way that is breaking even if we choose to not call it that.

Playing devil's advocate back on a breaking change: In my env everything is Ubuntu so why do I care that debian, centos, etc were different. You changed something very important and it should not be swept under the rug.

At the end of the day I leave it to you to evaluate what you think is best but I am torn on `Fixed` or `Breaking Change` I tend to always lean more breaking change because there is no real "cost" to moving to a higher version of software and prevents users from pulling in potentially unsafe changes.